pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "shell-functools"
+description = "A collection of functional programming tools for the shell."
+authors = [{ name = "David Peter", email = "mail@david-peter.de" }]
+urls = { Home = "https://github.com/sharkdp/shell-functools" }
+keywords = ["shell", "functional-programming", "filesystem", "string-manipulation", "command-line"]
+classifiers=[
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+readme = { file = "README.md", content-type = "text/markdown" }
+license = "MIT"
+requires-python = ">=3.5"
+dynamic = ["version"]
```

Comment:
Is there a reason not to make the `version` static, and remove the `version.py`?